sphinx/texinputs/latexmkjarc

```diff
@@ -0,0 +1,7 @@
+$latex = 'platex --halt-on-error --interaction=nonstopmode -kanji=utf8 %O %S';
+$dvipdf = 'dvipdfmx %O -o %D %S';
+$makeindex = 'mendex -U -f -d %B.dic -s python.ist %S || echo "mendex exited with error code $? (ignoring)" && : >> %D';
```

Comment:
@shimizukawa thanks for this info, I guess I have to plunge into the Perl examples coming with Latexmk to see how it handles things. Also as suggested by @tk0miya we can have per platform latexmkrc files if needed, but then the problem is I don't know offhand how to check from the Makefile the platform... (I am lacking basic software knowledge !)